### ISSUE TITLE
Training script

### ISSUE THREAD

**Issue by zzadiues3338** [simonMadec/VegAnn#5]
Hi, I want to fine tune this model on my own dataset.
Do we have a training script for training the VegAnn model ? 

Thanks a ton

**Comment by simonMadec** [comment]
Yes here is an example of how we train Vegann with pytorch lighting : 
https://github.com/simonMadec/VegAnn/blob/main/vegann/trainer.py

For fintening, you should load the model weights before or train by yourself on the VegAnn and then finetune!
